README.md

```diff
@@ -2,8 +2,6 @@
 Let's collect the data of Woman on GitHub, who inspired you to code more, learn more.
 
 **Note:** Please maintain the below list in alphabetical order.
-
-[alicetragedy](https://github.com/alicetragedy)
```

Comment:
@sara-02 Thanks for understanding :-) Please publish the commits. @tapasweni-pathak can review them once she gets a chance.